src/build.sh

```diff
@@ -19,7 +19,9 @@ PATHS="-Fu${SRC}html -Fi${SRC}html -Fi${SRC}html/entities.inc"
 DEFINES="-dUSEROPES -dLINES -dPARSEERROR"
 
 echo "Writing $VERSION_FILE"
-(git rev-list --count HEAD || echo "0") > "$VERSION_FILE"
+# If you update the fallback below also update WATTSI_LATEST in
+# https://github.com/whatwg/html-build/blob/master/build.sh
+(git rev-list --count HEAD || echo "76") > "$VERSION_FILE"
```

Comment:
```suggestion
(git rev-list --count HEAD || echo "78") > "$VERSION_FILE"
```

As far as I can tell, the right value here (and in the HTML build script) should be `78`.